Cargo.toml

```diff
@@ -89,3 +89,6 @@ codegen-units = 1
 # FIXME: LTO cannot be enabled for binaries in a workspace
 # <https://github.com/rust-lang/cargo/issues/9330>
 # lto = true
+
+[patch.crates-io]
```

Comment:
Can be removed.

Reply:
Would it make a difference if I comment out "[patch.crates-io]"?

Reply:
I guess it's fine but again: please add an explanation why this is kept around.